protostar-rust/src/lib.rs

```diff
@@ -42,9 +48,20 @@ fn internal_run_tests(input_path: &str) -> anyhow::Result<()> {
 }
 
 #[pyfunction]
-fn run_tests(input_path: String) -> PyResult<()> {
-    internal_run_tests(&input_path.to_owned())
-        .map_err(|e| PyErr::new::<RuntimeError, _>(format!("{:?}", e)))?;
+fn run_tests(input_path: String, contracts_paths_arg: &PyDict) -> PyResult<()> {
+  let mut contracts_paths = HashMap::new();
+  for key in contracts_paths_arg.keys() {
+    let paths: Vec<String> = contracts_paths_arg.get_item(key).expect("a key has to have a value").extract().unwrap();
+    if paths.len() > 1 {
+      return Err(PyValueError::new_err(format!("Cairo 1 allows only 1 path per contract name, {} detected for name {}", paths.len(), key)));
+    }
+    let path = paths[0].clone();
+    let key_converted: String = key.extract().unwrap();
+    contracts_paths.insert(key_converted, path);
+  }
+  let test_config = ProtostarTestConfig{ contracts_paths };
+    internal_run_tests(&input_path.to_owned(), test_config)
+        .map_err(|e| PyErr::new::<PyRuntimeError, _>(format!("{:?}", e)))?;
 
     Ok(())
 }
```

Comment:
I'm not sure about extensive use of `unwrap` in module like that, but we can resolve that in Rust CLI.